canvas_account_admin_tools/templates/canvas_account_admin_tools/base.html

```diff
@@ -8,8 +8,8 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Admin Tasks</title>
 
-        <link href="{% static 'bootstrap-3.3.4/css/bootstrap.min.css' %}" rel="stylesheet">
```

Comment:
So looks like we were pointing to an incorrect version for the last couple months..